map.js

```diff
@@ -171,67 +177,81 @@ const app = new Vue({
           [_.max(lng), _.max(lat)], // ne
         ], { padding: 10 });
 
+		var defaultIcon = 'star-15-red';
+
         // add the map layers to the map, and also to the vue mapLayers
         // data. Not currently using initiallyChecked.
         if (affiliateFalse.features.length) {
+  		  var icon = defaultIcon;
+  		  var iconImg = icon + ".svg";
           _this.map.addSource('marchon-affiliate-false-geojson', { type: 'geojson', data: affiliateFalse });
-          _this.addLayer('marchon-affiliate-false', 'marchon-affiliate-false-geojson', { 'icon-image': 'star-15-red' });
+          _this.addLayer('marchon-affiliate-false', 'marchon-affiliate-false-geojson', { 'icon-image': icon });
           _this.mapLayers.push({
             layerId: 'marchon-affiliate-false',
             label: 'Non Affiliates',
-            icon: 'star-15-red.svg',
+            icon: iconImg,
             initiallyChecked: true,
           });
         }
         if (affiliateTrue.features.length) {
+		  var icon = (isEventsPage ? defaultIcon : 'smallstar');
+		  var iconImg = icon + ".svg";
           _this.map.addSource('marchon-affiliate-true-geojson', { type: 'geojson', data: affiliateTrue });
-          _this.addLayer('marchon-affiliate-true', 'marchon-affiliate-true-geojson', { 'icon-image': 'smallstar' });
+          _this.addLayer('marchon-affiliate-true', 'marchon-affiliate-true-geojson', { 'icon-image': icon });
           _this.mapLayers.push({
             layerId: 'marchon-affiliate-true',
             label: 'Affiliates',
-            icon: 'smallstar.svg',
+            icon: iconImg,
             initiallyChecked: true,
           });
         }
         // familySepEvents
         if (familySepEventsPast.features.length) {
+  		  var icon = (isEventsPage ? defaultIcon : 'star-gray-light');
+		  var iconImg = icon + ".svg";
           _this.map.addSource('marchon-family-sep-events-past-geojson', { type: 'geojson', data: familySepEventsPast });
-          _this.addLayer('marchon-family-sep-events-past', 'marchon-family-sep-events-past-geojson', { 'icon-image': 'star-gray-light' });
+          _this.addLayer('marchon-family-sep-events-past', 'marchon-family-sep-events-past-geojson', { 'icon-image': icon });
           _this.mapLayers.push({
             layerId: 'marchon-family-sep-events-past',
             label: 'Family Separation Events (Past)',
-            icon: 'star-gray-light.svg',
+            icon: iconImg,
             initiallyChecked: true,
           });
         }
         if (familySepEventsFuture.features.length) {
+    	  var icon = (isEventsPage ? defaultIcon : 'star-blue');
```

Comment:
Family separation events shouldn't wind up on the events page, so no need for this